.eslintrc.js

```diff
@@ -62,6 +62,9 @@ module.exports = {
     'prefer-destructuring': 'warn',
     'no-param-reassign': 'warn',
 
+    // this should be fixed in header/headerA
+    'jsx-a11y/no-noninteractive-element-interactions': 'warn',
```

Comment:
I think we talked about this, but I can't remember well... Did this caused issues?